google-cloud-bigtable/lib/google/cloud/bigtable/client.rb

```diff
@@ -0,0 +1,39 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "google/cloud/bigtable/instance_set"
+
+module Google
+  module Cloud
+    module Bigtable
+      class Client
```

Comment:
How is this `Client` class different than the `InstanceAdminClient`, `TableAdminClient`, and `DataClient` objects that `Google::Cloud::Bigtable.new` returns?

How do users get an object? It looks like in the test setup you are calling `Google::Cloud::Bigtable::Client.new` directly instead of passing an argument to `Google::Cloud::Bigtable.new`. This seems incongruent.

Reply:
Client is a temporary class mostly just to write acceptance test. With Jiren working on the Bigtable class as entry point, this class will be removed and _instances_ function will be added to _Google::Cloud::Bigtable_